pkg/runner/step_context.go

```diff
@@ -508,44 +508,47 @@ func (sc *StepContext) runAction(actionDir string, actionPath string) common.Exe
 			stepID := 0
 			for _, compositeStep := range action.Runs.Steps {
 				stepClone := compositeStep
-                // Take a copy of the run context structure (rc is a pointer)
-                // Then take the address of the new structure
-                rcCloneStr := *rc
-                rcClone := &rcCloneStr
+				// Take a copy of the run context structure (rc is a pointer)
+				// Then take the address of the new structure
+				rcCloneStr := *rc
+				rcClone := &rcCloneStr
 				if stepClone.ID == "" {
 					stepClone.ID = fmt.Sprintf("composite-%d", stepID)
 					stepID++
 				}
-                rcClone.CurrentStep = stepClone.ID
+				rcClone.CurrentStep = stepClone.ID
 
-                if err := compositeStep.Validate(); err != nil {
-                    return err
-                }
+				if err := compositeStep.Validate(); err != nil {
+					return err
+				}
 
-                // Setup the outputs for the composite steps
-                if _, ok := rcClone.StepResults[stepClone.ID]; ! ok {
-                    rcClone.StepResults[stepClone.ID]  = &stepResult{
-                        Success: true,
-                        Outputs: make(map[string]string),
-                    }
-                }
+				// Setup the outputs for the composite steps
+				if _, ok := rcClone.StepResults[stepClone.ID]; !ok {
+					rcClone.StepResults[stepClone.ID] = &stepResult{
+						Success: true,
+						Outputs: make(map[string]string),
+					}
+				}
 
-				stepClone.Run = strings.ReplaceAll(stepClone.Run, "${{ github.action_path }}", filepath.Join(containerActionDir, actionName))
```

Comment:
Isn't this breaking `run` step using `${{ github.action_path }}`?

Reply:
I don't think so, although I'll need to check. I'm hoping that the context object's json mapping results in it still being available.